src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,6 +59,9 @@
 
     /**
      * Test body.
+     * @todo: #120 move constants in this file to their tests because tests
+     *  must share nothing
```

Comment:
@westonized my comment was about colon. I haven't spotted it before as I trusted green build. Time is optional indeed, but I'm just not sure what is the default value, so I'd recommend to specify it explicitly.   Puzzle validation task is somehow related. Build was green despite broken puzzle.